### PR TITLE
Update assign-licenses-to-user-accounts-with-microsoft-365-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Added another script as the previous one does work if there are multiple subscriptions and disabled plans in those subscriptions.

fixes #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10229

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell.md [hunk 1]
```diff
@@ -165,6 +165,51 @@ $addLicenses = @(
 Set-MgUserLicense -UserId "belinda@litwareinc.com" -AddLicenses $addLicenses -RemoveLicenses @()
 ```
 
+This example updates a user with **SPE_E5** (Microsoft 365 E5) and turns off the Sway and Forms service plans, while leaving the user's existing disabled plans in all other subscriptions in their current state:
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell.md]:
```suggestion
This example updates a user with **SPE_E5** (Microsoft 365 E5) and turns off the Sway and Forms service plans while leaving the user's existing disabled plans in all other subscriptions in their current state:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**